internals/plan/plan.go

```diff
@@ -822,6 +822,13 @@ func (layer *Layer) Validate() error {
 	return nil
 }
 
+// Layer implements the LayerCreator interface exposed by the plan state
+// manager. In this case it is simply a wrapper to make a already created
+// layer compatible with the interface.
+func (layer *Layer) Layer(planIn *Plan) (*Layer, error) {
+	return layer, nil
+}
```

Comment:
Make a layer, such as returned by ```ParseLayer``` compatible with ```LayerCreator``` on its own. This allows traditional use of a simple layer with AppendLayer and CombineLayer.